src/adapter/etl-adapter-doctrine/src/Flow/ETL/Adapter/Doctrine/SchemaConverter.php

```diff
@@ -159,13 +159,13 @@ private function flowToColumn(string $name, Type $type, ?Schema\Metadata $metada
         }
 
         $options = [
-            'notnull' => !$type->nullable(),
+            'notnull' => !$nullable,
         ];
 
         if ($type instanceof FloatType) {
             // with decimals precision and scale are confusing, in float precision is number of digits, not digits before/after decimal point
             // with decimals precision is total number of digits, and scale is number of digits after decimal point
-            $options['scale'] = $type->precision;
+            $options['scale'] = 6;
```

Comment:
I don't know the context but wasn't this change for testing purposes? If ok ignore

Reply:
Previously `type_float` had additional `precision` property that was used to round floats, which at the end of the day ended up with more complications so it was removed and rounding has become a user responsibility. 

I should remove it from here as well as scale can also be set by metadata 👍